src/olympia/devhub/templates/devhub/addons/submit/base.html

```diff
@@ -24,4 +24,21 @@ <h2 class="is_addon">{{ title }}</h2>
 <section class="addon-submission-process" role="main">
   {% block primary %}{% endblock %}
 </section>
+
+<div id="modals">
+  <div id="modal-confirm-submission-cancel" class="modal modal-confirm-submission-cancel">
+      <form method="POST" action="#">
+          <h3>{{ _('Cancel and Disable Version') }}</h3>
+          <p>
+              {% trans %}
+                Are you sure you wish to cancel your review request?
+              {% endtrans %}
+          </p>
+          <div class="modal-actions">
+              <button class="delete-button" type="submit">{{ _('Yes') }}</button>
+              {{ _('or') }} <a href="#" class="close">{{ _('Cancel') }}</a>
```

Comment:
I wouldn't bother with the `or` here: it's going to be weird in some languages to have the 3 words translated separately, might as well skip it.

Also, feels weird to be able to `cancel`... a `cancel` action. So maybe this should be instead:
```suggestion
              <button class="delete-button" type="submit">{{ _('Yes, Cancel') }}</button>
              <a href="#" class="close">{{ _('No, don't cancel') }}</a>
```